doc/build_instructions/windows_msvc.md

```diff
@@ -96,6 +126,24 @@ _Note:_ If you want to download and build Nyan automatically add `-DDOWNLOAD_NYA
     - Open a CMD window in `<openage directory>\build\` and run `python -m openage game`
     - Execute`<openage directory>\build\run.exe` every time after that and enjoy!
 
+_Note:_ you can just run this command in powershell to create a package folder with all the needed files
+```powershell
+mkdir package
+cd package
+mkdir dll
+cd ..
+$STAGING_PATH = Resolve-Path package
+$DLL_PATH = Join-Path package dll | Resolve-Path
+cd build
+$NYAN_DLL = (Get-ChildItem . -Recurse -Force -Filter 'nyan.dll')[0].FullName
+$OPENAGE_DLL = (Get-ChildItem . -Recurse -Force -Filter 'openage.dll')[0].FullName
+$NATIVE_OUTPUT = Split-Path -Path $OPENAGE_DLL -Parent
+Copy-Item -Path ./openage -Destination $STAGING_PATH -Recurse
+Copy-Item -Path $NYAN_DLL -Destination $DLL_PATH
+Copy-Item -Path (Join-Path $NATIVE_OUTPUT *.dll) -Destination $DLL_PATH
+Copy-Item -Path run.* -Destination $STAGING_PATH
+```
+
```

Comment:
if multiple commands are used, then you should put them into a scipt file

Reply:
may be a python script ?